client/src/Components/Pages/PatientQueue/PatientQueue.tsx

```diff
@@ -19,26 +19,26 @@ function PatientQueue() {
 
   const handleClick = (consultation: ConsultationInfo) => {
     updateCurrentConsultation(consultation);
-    history.push('/doctor/consultation');
-  }
+    history.push("/doctor/consultation");
+  };
 
   return (
-    <div>
     <div className="flex items-center justify-content-center flex-col bg-white-dark h-screen">
       <div className=" text-center mt-24 mb-10 px-3">
-      <h2 className="text-green text-3xl font-bold">
-          You have {consultations.getDoctorConsultations.length} patients waiting:
+        <h2 className="text-green text-3xl font-bold">
+          You have {consultations.getDoctorConsultations.length} patients
```

Comment:
Was it this line that was causing the doctor page to crash as the consultations may be undefined?